cli/create.go

```diff
@@ -202,6 +203,14 @@ func (r *RootCmd) create() *serpent.Command {
 				templateVersionID = template.ActiveVersionID
 			}
 
+			if len(templateVersion) > 0 {
+				version, err := client.TemplateVersionByName(inv.Context(), template.ID, templateVersion)
+				if err != nil {
+					return xerrors.Errorf("get template version by name: %w", err)
+				}
+				templateVersionID = version.ID
```

Comment:
This unfortunately does not accept the template version uuid. It would be better if the name or uuid worked.

We should fix this in the route handler imo: https://github.com/coder/coder/blob/main/coderd/templateversions.go#L885-L885

Not needed in this PR.